airflow/api_fastapi/core_api/routes/public/assets.py

```diff
@@ -276,3 +276,44 @@ def delete_dag_asset_queued_events(
 
     if result.rowcount == 0:
         raise HTTPException(status.HTTP_404_NOT_FOUND, f"Queue event with dag_id: `{dag_id}` was not found")
+
+
+@assets_router.get(
+    "/assets/queuedEvents/{uri:path}",
```

Comment:
```suggestion
    "/assets/queuedEvent/{uri:path}",
```

This change will be done in a follow up PR on all `queueEvent` endpoints to keep consistency.

Reply:
@pierrejeambrun is there an issue for that I can pick that up?

Reply:
Done